packaging/google-guest-agent.spec

```diff
@@ -36,14 +44,25 @@ Obsoletes: python-google-compute-engine, python3-google-compute-engine
 Contains the Google guest agent binary.
 
 %prep
+
+%if 0%{?build_plugin_manager}
+%autosetup -a 1
+%else
 %autosetup
+%endif
 
 %build
 for bin in google_guest_agent google_metadata_script_runner gce_workload_cert_refresh; do
   pushd "$bin"
   GOPATH=%{_gopath} CGO_ENABLED=0 %{_go} build -ldflags="-s -w -X main.version=%{_version}" -mod=readonly
   popd
 done
+# Build side-by-side both new agent (plugin manager) and legacy agent.
+%if 0%{?build_plugin_manager}
+pushd %{name}-extra-%{version}/
+  make cmd/google_guest_agent/google_guest_agent
```

Comment:
make accepts version now, can we pass `%{_version}` here as well?

Reply:
indeed. done.